spec/unit/inventory_spec.rb

```diff
@@ -51,4 +51,52 @@
       expect(resource.children_by_type('type-03', true).size).to eq(1)
     end
   end
+
+  describe '#ResultFetcher' do
```

Comment:
If this is a subclass of the inventory API, you can just do `describe ResultFetcher`. `#something` is used to denote instance methods.